database_refresh/adhoc-db-refresh-job.yaml

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
I believe you can always run a cronjob adhoc, ahead of scheduled, with this kube command. We could try, if it works there is no need for this additional file and we reduce duplication of code:

`kubectl -n <namespace> create job --from=cronjob/<name of cronjob> <name of job>`

So in this case _I think_ this should work:

`kubectl -n hmpps-complexity-of-need-production create job --from=cronjob/db-refresh-job adhoc-db-refresh`

Reply:
yes, that should work, we do this in CVL a lot - I just copied most of this directly over from another project and used this to test the job.  I'll remove

Reply:
removed adhoc job